subsys/mgmt/Kconfig.mcumgr

```diff
@@ -24,20 +24,74 @@ menuconfig MCUMGR_CMD_FS_MGMT
 	  Enables mcumgr handlers for file management
 
 if MCUMGR_CMD_FS_MGMT
+
+choice FS_MGMT_MAX_FILE_SIZE
+	prompt "Maximum file size that could be uploaded/downloaded"
+	default FS_MGMT_MAX_FILE_SIZE_64KB
+	default	FS_MGMT_MAX_FILE_SIZE_4GB
```

Comment:
is this needed? The firs choice default always take precedence. Doesn't it?

Reply:
Yes, this is mistake.